### DISCUSSION TITLE
Schedule all meeting within CAMARA anchored on UTC

### DISCUSSION THREAD

**Discussion by hdamker** [camaraproject/Governance#192] [Announcements]
**Update**: Proposal was accepted and decided by TSC on October 17th. See also https://github.com/camaraproject/Governance/issues/164#issuecomment-2578635106

**Problem description**

Currently most (all?) community meetings within CAMARA are scheduled anchored on the Central European Timezone, which means that the meetings are moving relative to UTC (Coordinated Universal Time) with the start and end of the daylight saving time (DST) period in EU.

In other words, a meeting which is scheduled within LFX platform on "16:00 Europe/Amsterdam" happens on
* CET 16:00 / UTC 15:00 from last Sunday in October to last Sunday in March
* CEST 16:00 / UTC 14:00 from last Sunday in March to last Sunday in October

That is convenient for meeting participants in Europe, but unfair and confusing for people within time zones without DST or on the southern hemisphere (see https://en.wikipedia.org/wiki/Daylight_saving_time_by_country - a lot countries had formerly DST observed but discontinued it). The meetings are jumping in time for them just because EU hasn't yet managed to discontinue DST.

North America is mostly in sync but has chosen second Sunday in March to enter DST and first Sunday in November to end DST, which adds also here a lot of confusion.

**Expected action**

**Proposed change - to be discussed and decided by the TSC:**
* **All CAMARA Community meetings get anchored in UTC**
* **Each community decides on which UTC time they want to anchor their meeting**
  * ~Default is the time during the CEST period, so in the example above UTC 15:00~ (edited)
  * Default is the time during the **CET** period, so in the example above UTC 15:00
 

Advantages: 
* clear schedule for all countries without DST periods
* countries with DST period are aware of their date of change and shouldn't be surprised

**Alternative Solutions**

The anchoring within EU time zone will be kept. To avoid misunderstanding, all descriptions of meeting schedules will be reviewed and updated if not clearly defined.

Possible example (edit: corrected):
```
Schedule: 16:00 CET / 15:00 UTC in winter and 16:00 CEST / 14:00 UTC during Central European Summer Time (last Sunday in March to last Sunday in October)
```
That will not solve that it is unfair to impose the time change on other countries without DST.

**Additional context**

**Comment by RegCox** [comment]
Fully support this change

**Comment by Kevsy** [comment]
Agreed - it's about time :grin:

**Comment by hdamker** [comment]
This complete time zone and DST thing is so confusing that I did it in the above proposal already in two place wrong, now edited and hopefully correct. 

As "authoritative" source here from [Wikipedia](https://en.wikipedia.org/wiki/Central_European_Summer_Time#:~:text=Central%20European%20Summer%20Time%20(CEST,other%20part%20of%20the%20year.):

> Central European Summer Time (CEST, UTC+02:00), sometimes referred to as Central European Daylight Time (CEDT), is the standard clock time observed during the period of summer daylight-saving in those European countries which observe Central European Time (CET; UTC+01:00) during the other part of the year.

**Comment by Kevsy** [comment]
Regarding ways to show the participant the local time for the meeting - here's an example of a native script to convert the UTC meeting time to the locale of the requesting browser/OS: 
[UTC Meeting Time to Local Time](https://kevsy.github.io/local-meeting-time.html?utc-meeting-time=2200) .

Unfortunately there is no way to embed and execute `<script>` blocks in GitHub README.MD files, but it could be included as a clickable link in the README files, and/or embedded in the camaraproject.org pages to dynamically show the local meeting time. 

Caveat: it works based on the time offset *now*, it doesn't take the date of a future meeting into account, so caution needed in March/October.

**Comment by diegogonmar** [comment]
OK to the proposal, thanks

**Comment by hdamker** [comment]
Changed the title of issue according to the decision within TSC on October 17th, 2024 ([meeting minutes](https://lf-camaraproject.atlassian.net/wiki/x/R4BwAQ)) 
- All CAMARA Community meetings get anchored in UTC
- Each community decides on which UTC time they want to anchor their meeting 

Keeping the issue open as reference for all communities within CAMARA. Please act accordingly and send your scheduling requests to support@camaraproject.org.

**Comment by hdamker** [comment]
Moving the issue to discussion (as announcement) to keep the information there.
